properties.schema

```diff
@@ -19,20 +19,12 @@
                 "_isEnabled": {
                   "type":"boolean",
                   "required":true,
+                  "default": true,
                   "title":"Is Enabled",
                   "inputType": {"type": "Boolean", "options": [true, false]},
                   "validators": [],
                   "help": "If set to 'true', this course will be tracked by TinCan/xAPI"
                 },
-                "_activityID": {
```

Comment:
I thought the activityID is passed in via the CMI5 method (via URL param) on launch of the course?